x-pack/plugins/uptime/public/components/monitor/ping_list/ping_redirects.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import { EuiIcon, EuiLink, EuiSpacer, EuiText } from '@elastic/eui';
+import { Ping } from '../../../../common/runtime_types/ping';
+
+const StyledLink = styled(EuiLink)`
+  margin-right: 5px;
+  margin-left: 5px;
+`;
+
+interface Props {
+  monitorStatus: Ping | null;
+  showDescription?: boolean;
+}
+
+export const PingRedirects: React.FC<Props> = ({ monitorStatus, showDescription }) => {
```

Comment:
Taking the entire `monitorStatus` is making testability here a bit rough. It's forcing tests to pass in a large complex object when only a few fields are needed.